test/blackbox-tests/test-cases/install/install-glob/install-glob-relative.t

```diff
@@ -43,6 +43,14 @@ Incorrect install stanza that would place files outside the package's install di
   stanzas beginning with .. will be disallowed to prevent a package's installed
   files from escaping that package's install directories.
 
+Test that we can disable the warning:
+  $ cat >>dune-project <<EOF
+  > (warnings (escaping_paths_in_install_stanza disabled))
+  > EOF
+  $ dune build foo.install
+
+Now we remove disabling the warning
+  $ sed -i '$d' dune-project
```

Comment:
The sed on macos doesn't allow `-i` without an extension to use for backup files. Use `sed -i.bak ...`.